src/i18n/strings/en_EN.json

```diff
@@ -201,7 +201,7 @@
         "failed_query_registration_methods": "Unable to query for supported registration methods.",
         "failed_soft_logout_auth": "Failed to re-authenticate",
         "failed_soft_logout_homeserver": "Failed to re-authenticate due to a homeserver problem",
-        "footer_powered_by_matrix": "powered by Matrix",
+        "footer_powered_by_matrix": "powered by Sociail",
```

Comment:
It does seem like whatever this is, it probably is still powered by Matrix ... ?